src/productcatalogservice/Dockerfile

```diff
@@ -17,7 +17,7 @@ FROM alpine AS release
 
 WORKDIR /usr/src/app/
 
-COPY ./src/productcatalogservice/products.json ./
+COPY ./src/productcatalogservice/products/ ./products/
```

Comment:
I've played around with this and I believe it would be easier to simply add the products as a volume. WDYT?

Reply:
With this we can modify the products without the need to re-build the container image.

Reply:
This approach (mounting as a volume) would make the Helm deployment kinda messy as we would also need to mount it as a volume + config map in Helm.

Reply:
hmmmm, but how are we handling this?
https://github.com/open-telemetry/opentelemetry-demo/blob/main/docker-compose.yml#L562-L563

I couldn't find it on the Helm charts

Reply:
> hmmmm, but how are we handling this? https://github.com/open-telemetry/opentelemetry-demo/blob/main/docker-compose.yml#L562-L563
> 
> I couldn't find it on the Helm charts

That file is empty and meant to be used as an example. The Helm chart doesn't have a default for it, but if you wanted to update the feature flags, you would use the same `mountedConfigMaps` option from Helm as we would do to add additional products.

Making this a default would cause some messiness in Helm (arrays don't merge), but leaving it as an optional thing makes it easier to manage.

Reply:
Got it!
Thanks @puckpuck! 🚀